x-pack/plugins/lens/public/indexpattern_plugin/loader.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// import { Chrome } from 'ui/chrome';
+import chrome from 'ui/chrome';
```

Comment:
This won't work in the new platform world. The shimmed plugin (`x-pack/plugins/lens/public/indexpattern_plugin/plugin.tsx`) should do this import and pass it down to the loader